Translated some boss tips.

Sonickyle27 opened the pull request:
I translated boss tips up to Varda Soma. I didn't translate Dead_TransformMechaA_004#0 as I'm not sure about the translation.

It's sorta a Google Translate translation, but I did check boss names and videos to ensure that they were correct.

Edit: Implemented @SynthSy's corrections. I also translated Dead_BossMechaA_004#0, which I missed. oops.

Commit:
Translated some boss tips.

I translated the Boss tips up to Vardha Soma. I'm not sure about Dead_TransformMechaA_004#0 so I didn't translate it.

Commit:
Update ui_tips.csv

Commit:
Translated some boss tips.

I translated up to Vardha Soma. I left Dead_TransformMechaA_004#0 because I'm not sure about the translation. I did three commits to fix my spelling mistakes darn it.

Commit:
Spelling.

Commit:
Fixed some line breaks in my translated boss tips.

<br< won't work.

SynthSy commented on UI/ui_tips.csv:
```diff
@@ -309,11 +309,11 @@
 01_MainLobby_121#1,"""“PS Vita”専用ブロックでウィークリータイムアタックに挑戦すると<br>“PS Vita”専用のランキングにエントリーされる。"""
 01_MainLobby_133#1,"""クイックメニューの表示内容は、<br><pict Sign_TRIANGLE>でパターンを変更できる。"""
-Dead_BigBeastA_001#0,"""ロックベアは顔が弱点だ。ロックオンをうまく利用して攻撃しよう。"""
+Dead_BigBeastA_001#0,"""The Rockbear's face is a weak point.<br>If you lock onto the face, you can deal increased damage."""
```
Don't add extra fluff such as "If you lock onto the face, you can deal increased damage."

Correct usage of the sentence would be "Try to lock on to its face and attack it!"

SynthSy commented on UI/ui_tips.csv:
```diff
@@ -392,22 +392,22 @@ Dead_DragonSummoner_002#0,"""When the Goronzoran raises its stick, it's about to
-Dead_TransformMechaA_001#0,"""トランマイザーは両腕で叩きつける攻撃の後、<br>弱点のコアが露出する。背後を狙って攻撃しよう。"""
-Dead_TransformMechaA_002#0,"""トランマイザーのミサイル攻撃はこちらを追いかけてくる。<br>攻撃を行わず、移動し続けて回避しよう。"""
-Dead_TransformMechaA_003#0,"""トランマイザーはミサイルによる攻撃の後、<br>弱点のコアが露出する。ミサイルを回避したら攻撃のチャンスだ。"""
+Dead_DragonGladiator_001#0,"""When Dragon Ex lifts both of their arms at the same time, it is preparing a<br>powerful attack. Run to the side of the Dragon Ex to avoid the attack."""
```
Dragon Ex is a singular enemy, therefore the use of "its" instead of "their" would be optimal.

SynthSy commented on UI/ui_tips.csv:
```diff
@@ -392,22 +392,22 @@ Dead_DragonSummoner_002#0,"""When the Goronzoran raises its stick, it's about to
-Dead_TransformMechaA_003#0,"""トランマイザーはミサイルによる攻撃の後、<br>弱点のコアが露出する。ミサイルを回避したら攻撃のチャンスだ。"""
+Dead_DragonGladiator_001#0,"""When Dragon Ex lifts both of their arms at the same time, it is preparing a<br>powerful attack. Run to the side of the Dragon Ex to avoid the attack."""
+Dead_DragonGladiator_002#0,"""When Dragon Ex stabs their sword into the ground, a shard of light pierces from the ground. You can keep running to avoid this."""
+Dead_DragonGladiator_003#0,"""Dragon Ex's shields on its arms can be destroyed.<br>If they are destroyed, the Dragon Ex's swords won't reach as far."""
```
This sentence feels off. Consider rewriting this to sound better.

SynthSy replied:
Also, the last bit is incorrect. 

> 破壊すると、剣を使用した攻撃時に大きな隙が生まれる。

This refers to creating a gap once you've broken the parts.

Sonickyle27 replied:
Which of the sentences are you referring to?

SynthSy replied:
Click on "Files changed" at the top, and you should see what I'm referring to.

SynthSy commented on UI/ui_tips.csv:
```diff
@@ -392,22 +392,22 @@ Dead_DragonSummoner_002#0,"""When the Goronzoran raises its stick, it's about to
+Dead_DragonGladiator_003#0,"""Dragon Ex's shields on its arms can be destroyed.<br>If they are destroyed, the Dragon Ex's swords won't reach as far."""
+Dead_DragonGladiator_004#0,"""Poison is extremely effective against Dragon Ex's head, shields, and tail.<br>When using poison, it's easy to do lots of damage to break these parts."""
+Dead_TransformMechaA_001#0,"""After Transmizer slams the ground with both arms, its core is revealed.<br>You can attack this weak point to deal additional damage."""
+Dead_TransformMechaA_002#0,"""Transmizer's missiles try to home in on you. If you keep moving,<br>, you can avoid the missiles."""
```
Consider rewriting this sentence.

SynthSy commented on UI/ui_tips.csv:
```diff
@@ -392,22 +392,22 @@ Dead_DragonSummoner_002#0,"""When the Goronzoran raises its stick, it's about to
-Dead_BossMechaA_003#0,"""ビッグヴァーダーの甲板に登った後は、後方のミサイルポッドと<br>本体肩のミサイルポッドを破壊しよう。"""
+Dead_BossMechaA_001#0,"""You should destroy the Big Vardha's guns on the sides before climbing to the deck."""
+Dead_BossMechaA_002#0,"""The flamethrowers behind Big Vardha are dangerous when attacking from the front.<br>Try attacking them from the side."""
+Dead_BossMechaA_003#0,"""The Big Vardha has missile arrays that are used to shoot you when<br>you are on the deck. You should destory them to stop the missiles."""
```
> ビッグヴァーダーの甲板に登った後は、後方のミサイルポッドと<br>本体肩のミサイルポッドを破壊しよう。

Talks about destroying the missile pods on the rear and sides of Vardha.

SynthSy commented on UI/ui_tips.csv:
```diff
@@ -392,22 +392,22 @@ Dead_DragonSummoner_002#0,"""When the Goronzoran raises its stick, it's about to
-Dead_MachineLittleBoss_003#0,"""ヴァーダーソーマのメインブースターが青白く光ったら電撃攻撃の合図。<br>急いで離れて攻撃を回避しよう。"""
-Dead_MachineLittleBoss_004#0,"""ヴァーダーソーマは主砲からビームを放ちながら1回転する。<br>攻撃範囲は広いが、足元は安全なので近づいて回避しよう。"""
+Dead_MachineLittleBoss_001#0,"""Vardha Soma has a main booster and three sub-boosters. If you destroy the sub-boosters,<br>you can stop the Vardha Soma attacking from the air."""
+Dead_MachineLittleBoss_002#0,"""A lot of Varha Soma's parts can be destroyed. Aim to damage the three sub-boosters,<br<the two shoulder missile arrays, as well as the secondary and main guns."""
```
This entire sentence focuses on what you can destroy. It does not tell you about the boosters, but only the front cannon and the mini turrets.

SynthSy commented on UI/ui_tips.csv:
```diff
@@ -392,22 +392,22 @@ Dead_DragonSummoner_002#0,"""When the Goronzoran raises its stick, it's about to
-Dead_MachineLittleBoss_004#0,"""ヴァーダーソーマは主砲からビームを放ちながら1回転する。<br>攻撃範囲は広いが、足元は安全なので近づいて回避しよう。"""
+Dead_MachineLittleBoss_001#0,"""Vardha Soma has a main booster and three sub-boosters. If you destroy the sub-boosters,<br>you can stop the Vardha Soma attacking from the air."""
+Dead_MachineLittleBoss_002#0,"""A lot of Varha Soma's parts can be destroyed. Aim to damage the three sub-boosters,<br<the two shoulder missile arrays, as well as the secondary and main guns."""
+Dead_MachineLittleBoss_003#0,"""When the Vardha Soma's main booster glows pale-white, it's about to create an electrical shockwave. Run away when you see this."""
```
青白く, Adjective for pale, bluish white.

SynthSy commented on UI/ui_tips.csv:
```diff
@@ -392,22 +392,22 @@ Dead_DragonSummoner_002#0,"""When the Goronzoran raises its stick, it's about to
+Dead_MachineLittleBoss_001#0,"""Vardha Soma has a main booster and three sub-boosters. If you destroy the sub-boosters,<br>you can stop the Vardha Soma attacking from the air."""
+Dead_MachineLittleBoss_002#0,"""A lot of Varha Soma's parts can be destroyed. Aim to damage the three sub-boosters,<br<the two shoulder missile arrays, as well as the secondary and main guns."""
+Dead_MachineLittleBoss_003#0,"""When the Vardha Soma's main booster glows pale-white, it's about to create an electrical shockwave. Run away when you see this."""
+Dead_MachineLittleBoss_004#0,"""When the Vardha Soma is spinning and shooting its laser from the main gun,<br>the laser won't hit you if you stand under Vardha Soma."""
```
For starters, this explains that when Soma starts to spin, it makes a full rotation once. During this time, it's best to keep on your toes (or on your feet, aka move around).

While this is what most people do when fighting Soma, this is not what the tip says.

SynthSy reviewed (changes requested):
Some changes need to be addressed and performed before one should merge this.

Other than that, it's ok.

Commit:
Implemented @SynthSy's corrections

With the Varda Soma, it has "3" so idk what that's referring to that isn't sub-boosters.
